syftlib/src/main/java/org/openmined/syft/networking/clients/RetryPolicy.kt

```diff
@@ -0,0 +1,17 @@
+package org.openmined.syft.networking.clients
+
+const val DEFAULT_RETRY_DELAY = 2000L
+const val DEFAULT_MAX_RETRY_COUNT = 3
+const val DEFAULT_RETRY_DELAY_FACTOR = 2
+
+data class RetryPolicy(

```

Comment:
This class might come in handy if we have it's use case somewhere else as well. Do you have some places in mind?